contributors.md

```diff
@@ -418,23 +418,23 @@ I also hope that it can convince people who don't consider themselves capable of
 
 ### Benjamin Mummery
 
-* Role: Infrastructure Maintainer (2023 - present)
-* GitHub id: [bsipocz](https://github.com/bsipocz)
-* ORCID: [0000-0002-3713-6337](https://orcid.org/0000-0002-3713-6337)
+* Twitter: [@BenjaminMummery](https://twitter.com/@BenjaminMummery)
 
 * Short bio:
 > While studying Astrophysics, Benjamin was the first speaker at, and later organizer of, Liverpool PubHD ([Facebook](https://www.facebook.com/pubhdliverpool/), [Twitter](https://twitter.com/pubhd_liverpool?lang=en-gb)) - a monthly cross-discipline event that challenged PhD students to "explain their research in 10 minutes, while enjoying a pint." During this time he also developed and delivered multiple more official outreach events including presenting to both the public and airforce commanders at [RAF Cosford Airshow](https://www.youtube.com/watch?v=KImlYiCO7Vs), and frequent activities for schools. Now a Research Software Engineer at the Hartree Centre, he has continued his engagement with outreach, most recently in collaboration with Tim Powell designing a LEGO version of the Centre's iconic supercomputer [Scafell Pike](https://twitter.com/HartreeCentre/status/1189145621564968963). He can frequently be found banging on about [fictional](https://www.youtube.com/watch?v=zrqfT4aotfE) [time travel](https://inews.co.uk/culture/film/donnie-darko-reviewed-astrophysicist-531742).
 
+* Personal highlights:
+> I collaborated with a group of researchers from The Alan Turing Institute to draft a chapter of scientific outreach.
+
 ### Brigitta Sipőcz
 
-* Twitter: [@BenjaminMummery](https://twitter.com/@BenjaminMummery)
+* Role: Infrastructure Maintainer (2023 - present)
+* GitHub id: [bsipocz](https://github.com/bsipocz)
+* ORCID: [0000-0002-3713-6337](https://orcid.org/0000-0002-3713-6337)
+
 * Short bio:
 > I am an astronomer turned Research Software Engineer. I work at Caltech/IPAC to build and improve tools, e.g. Python libraries and Science Platforms to provide ways to access data in the NASA/IPAC Infrared Science Archive. Prior joining IPAC I was DiRAC Fellow in the data engineering team at the Institute for Data Intensive Research in Astrophysics and Cosmology in Seattle. I am a developer and maintainer of several open source astronomy libraries and their infrastructure (e.g. astroML, astroquery, astropy) and I enjoy contributing to upstream projects as well. I have a keen interest in finding ways to promote the open development model and make tools more sustainable. I am a fellow of the Software Sustainability Institute.
 
```

Comment:
GH didn't let me add a suggestion on the deleted lines, so I added it again. Feel free to edit/fix grammar/etc as needed.

```suggestion

* Personal highlights:
> Having the opportunity to work with graphic designers during the Book Dash events. It was fascinating to witness the iterations and evolution from (occasionally) fuzzy ideas into gorgeous illustrations. I keep using the material outside of TTW for my own talks about open source and collaboration.  
> The small zoom celebration we had with the infrastructure WG team as we migrated to The Turing Way github organization. 
```